docusaurus/docs/getting-started/what-s-included/environment-variables.md

```diff
@@ -0,0 +1,57 @@
+---
+id: environment-variables
+title: Environment Variables
+sidebar_label: Environment Variables
+---
+
+This boilerplate is setup such that environment variables are passed at buildtime.
+You can access these variables from `process.env`, but all variables must be defined before they can be used.
+
+First, you have to define them in your `.env` file.
+We already provide an `.env.sample` file, which you can clone to a `.env` file so you have a starting point with all the environment variables.
+
+The following example is defining the `GA_TRACKING_ID` variable:
+
+```bash
+# Google Analytics Tracking Code
+GA_TRACKING_ID=12345
+```
+
+If you want to add a environment variable, you must define them in the same way:
+
+```bash
+# Google Analytics Tracking Code
+GA_TRACKING_ID="12345"
```

Comment:
Hmmm, one has quotations but the other doesn't.

Reply:
Done ✅only added quotations for the content that has spaces in it.